### DISCUSSION TITLE
[Feature] Allow Display Order To Be Oldest First For Shared Links

### DISCUSSION THREAD

**Discussion by Menghini** [immich-app/immich#8123] [Feature Request]
### The feature

Most of my users are expecting the photos to be oldest first. The new option added in v1.99.0 is great, but it is 1) not allowed to be changed in a shared album and 2) I would like to have the oldest first option be the default for a shared album, or at the very least, allow the person sharing the link to dictate which order will be default when a user visits from a shared link.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by Kixel** [comment]
I'd love this as well. Just had to search around for an alternative to Immich just because this was not a feature.
Please at least just use the order that's set in the display order settings for the person sharing the album!

**Comment by sym0nd0** [comment]
I've just come to use the feature for the first time, to share our wedding photos. I love it, but in their current order it looks like we were married and then my wife moonwalked out the chapel away from me. 😂

I thought it would follow the same order setting from the user that created the link (me, with the Oldest photo first). I would really like to have the option to be able to set the display order when creating the sharing link.

**Comment by bo0tzz** [comment]
Now supported with #10528
